Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -0,0 +1,139 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class EditorialProjectsViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var closeButton: UIButton = {
+    UIButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var discoveryPageViewController: DiscoveryPageViewController = {
+    DiscoveryPageViewController.configuredWith(sort: .newest)
+      |> \.preferredBackgroundColor .~ .clear
```

Comment:
Setting this here and not in `bindStyles` because the property must be set when the VC is instantiated so that it will be available at the time that `bindStyles` is called in `DiscoveryPageViewController`.